component/service/src/main/java/org/exoplatform/social/rest/impl/activity/ActivityRestResourcesV1.java

```diff
@@ -689,7 +689,7 @@ public Response shareActivity(
                                 @QueryParam("expand")
                                 String expand,
                                 @RequestBody(description = "Share target spaces", required = true)
-                                ActivityEntity model) {
+                                ExoSocialActivityImpl model) {
```

Comment:
Could you use ActivityEntity and add needed property there if necessary.
What is the difference between summary and body of an activity ?

Reply:
in news entity has a title, summary and body 
in ExoSocialActivityImpl i can get all information about shared activity  but ActivityEntity not contain summary for shared activity to passed in email template